diff --git a/client/CMakeLists.txt b/client/CMakeLists.txt
index 1fbc84e..8e2d654 100644
--- a/client/CMakeLists.txt
+++ b/client/CMakeLists.txt
@@ -1,17 +1,8 @@
 set_app_name( PROGNAME qdigidoc4 )
 
-add_executable( TSLDownload TSLDownload.cpp )
-target_link_libraries( TSLDownload Qt5::Network )
-add_custom_command(
-	OUTPUT TSL.qrc
-	DEPENDS TSLDownload
-	COMMAND $<TARGET_FILE:TSLDownload> "${CMAKE_CURRENT_BINARY_DIR}" ${TSL_URL} ${TSL_INCLUDE}
-	WORKING_DIRECTORY ${_qt5Core_install_prefix}/bin
-)
-
 configure_file( translations/tr.qrc tr.qrc COPYONLY )
 qt5_add_translation( SOURCES translations/en.ts translations/et.ts translations/ru.ts )
-qt5_add_resources( SOURCES images/images.qrc fonts/fonts.qrc ${CMAKE_CURRENT_BINARY_DIR}/tr.qrc ${CMAKE_CURRENT_BINARY_DIR}/TSL.qrc )
+qt5_add_resources( SOURCES images/images.qrc fonts/fonts.qrc ${CMAKE_CURRENT_BINARY_DIR}/tr.qrc TSL.qrc )
 qt5_wrap_ui( SOURCES
 	MainWindow.ui
 	dialogs/AddRecipients.ui
@@ -313,25 +304,25 @@ else()
 	configure_file( qdigidoc4.1.cmake qdigidoc4.1 )
 	install( TARGETS ${PROGNAME} DESTINATION ${CMAKE_INSTALL_BINDIR} )
 	install( FILES ${CMAKE_CURRENT_BINARY_DIR}/qdigidoc4.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1 )
-	install( FILES qdigidoc4.desktop DESTINATION ${CMAKE_INSTALL_DATADIR}/applications )
-	install( FILES qdigidoc4.xml DESTINATION ${CMAKE_INSTALL_DATADIR}/mime/packages )
+	install( FILES qdigidoc4.desktop DESTINATION ${CMAKE_INSTALL_DATADIR}/applications RENAME ee.ria.qdigidoc4.desktop )
+	install( FILES qdigidoc4.xml DESTINATION ${CMAKE_INSTALL_DATADIR}/mime/packages RENAME ee.ria.qdigidoc4.xml )
 	foreach(RES 16 22 32 48 128)
 		foreach(TGT application-vnd.etsi.asic-e+zip.png application-vnd.etsi.asic-s+zip.png application-x-ddoc.png application-vnd.lt.archyvai.adoc-2008.png)
 			install(
 				FILES images/qdigidoc_client_document_${RES}x${RES}.png
 				DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/${RES}x${RES}/mimetypes/
-				RENAME ${TGT}
+				RENAME ee.ria.qdigidoc4.${TGT}
 			)
 		endforeach()
 		install(
 			FILES images/qdigidoc_crypto_document_${RES}x${RES}.png
 			DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/${RES}x${RES}/mimetypes/
-			RENAME application-x-cdoc.png
+			RENAME ee.ria.qdigidoc4.application-x-cdoc.png
 		)
 		install(
 			FILES images/digidoc_icon_${RES}x${RES}.png
 			DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/${RES}x${RES}/apps/
-			RENAME qdigidoc4.png
+			RENAME ee.ria.qdigidoc4.png
 		)
 	endforeach()
 endif()
