--- DigiDoc4-Client/client/QPKCS11.cpp
+++ DigiDoc4-Client/client/QPKCS11-fix.cpp
@@ -365,7 +365,7 @@ bool QPKCS11::reload()
 #elif defined(Q_OS_WIN)
 		{ "opensc-pkcs11.dll", {} },
 #else
-		{ "opensc-pkcs11.so", {} },
+		{ "/app/lib/opensc-pkcs11.so", {} },
 #if defined(Q_OS_LINUX)
 		{ "/opt/latvia-eid/lib/eidlv-pkcs11.so", "3BDD18008131FE45904C41545649412D65494490008C" }, // LV-G1
 		{ "/opt/latvia-eid/lib/eidlv-pkcs11.so", "3BDB960080B1FE451F830012428F536549440F900020" }, // LV-G2
